src/service/angular-sails.js

```diff
@@ -10,7 +10,7 @@ angular.module('ngSails').provider('$sails', function () {
     this.responseHandler = undefined;
 
     this.$get = ['$q', '$timeout', function ($q, $timeout) {
-        var socket = io.connect(provider.url),
+        var socket = io.sails.connect(provider.url),
```

Comment:
Can you make this backward compatible with something like `(io.sails || io).connect(provider.url)`?